mainwindow.cc

```diff
@@ -763,6 +763,10 @@ MainWindow::MainWindow( Config::Class & cfg_ ):
   connect( &articleNetMgr, SIGNAL( proxyAuthenticationRequired( QNetworkProxy, QAuthenticator * ) ),
            this, SLOT( proxyAuthentication( QNetworkProxy, QAuthenticator * ) ) );
 
+  QNetworkDiskCache *diskCache = new QNetworkDiskCache(this);
+  diskCache->setCacheDirectory(QStandardPaths::writableLocation(QStandardPaths::CacheLocation));
+  articleNetMgr.setCache(diskCache);
```

Comment:
When a Wikipedia article is already cached, these three lines of code actually reduce the amount of sent and received network data almost tenfold! The loading time reduction is particularly noticeable when there are large images in an article (this is the case in many Wookieepedia articles in my #966 branch).

I am going to create another pull request with this code, but in addition clear the cache on exit in the portable version as @Abs62 suggested in https://github.com/goldendict/goldendict/issues/1279#issuecomment-716291835. Thank you @nikita-moor for this great idea!